src/generate-services.ts

```diff
@@ -21,9 +21,6 @@ import SourceInfo, { Fields } from "./sourceInfo";
 import { contextTypeVar } from "./main";
 import { Context } from "./context";
 
-const hash = imp("hash*object-hash");
-const dataloader = imp("DataLoader*dataloader");
```

Comment:
Fwiw, this is admittedly kind of cute, but if you'd like how I've handled this in the past is by having ts-poet rewrite the imports.

That was more useful when the imports were scattered all over the ts-proto codebase, while granted for these imports, they're only used ~once/twice so it's not that bad.

That said, if you'd like to try it, I just pushed a new version of ts-poet to handle default --> star rewrites (https://github.com/stephenh/ts-poet/commit/29c834db9409ff7c4fdc2440a2d0e1d75f6de2bc) so, in theory you could:

* Bump ts-poet in package.json to 6.4.1
* Keep these consts by as `const hash = imp("hash=object-hash");
* Update `getTsPoetOpts` to include `dataloader` and `object-hash` in the `imports` list, which should a) leave them as-is if `esModuleInterop` is on, or b) rewrite them to the old `import ... as *` if `esModuleInterop` is _not_ on

Do you mind trying that and seeing if it works?